__tests__/inputContainer.test.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import { render, fireEvent } from '@testing-library/react';
+
+import { useSelector, useDispatch } from 'react-redux';
+
+import InputContainer from '../src/containers/InputContainer';
+
+jest.mock('react-redux');
+
+test('InputContainer', () => {
+  const dispatch = jest.fn();
+  useDispatch.mockImplementation(() => dispatch);
```

Comment:
```suggestion
  const dispatch = jest.fn();
  
  useDispatch.mockImplementation(() => dispatch);
```

얘네 둘은 별개입니다ㅎㅎ

심지어 useDispatch.mockImplementation은 beforeEach 훅 안에서 쓰일 때가 많죠.

Reply:
@wholemann
적용 완료했습니다.